### DISCUSSION TITLE
BLEAlias does not work after reboot (ESP32)

### DISCUSSION THREAD

**Discussion by jakrzy** [arendst/Tasmota#15408] [Q&A] [answered]
The BLEAlias setting entered from the console is not remembered after restart.
I tried to set up in a script
> B
=> BLEAlias xxxxxx = flora
however, after tasmot restarts, the alias is not set.
Other settings in the> B section, such as => Power1 0, are set correctly.
ESP32-dev tasmota-dev 04/19/2022
Regards Jakrzy

**Comment by barbudor** [comment, marked as answer]
Most BLE and MI32 settings are not saved to Flash Settings
You can set at every boot using a boot rule or an autoexec.bat file which is the most convenient
Go to Consoles => Manage File System
Use the button to create and edit a new file that you will name autoexec.bat
Enter each BLEAlias or Mi32Option5 command one per line
Save and restart

**Reply by jakrzy** [reply]
Thank you, it works :)
Regards Jacek
